packages/enzyme/src/ShallowWrapper.js

```diff
@@ -973,7 +982,7 @@ class ShallowWrapper {
    */
   parents(selector) {
     return this.single('parents', (n) => {
-      const allParents = this.wrap(this.single('parents', n => nodeParents(this, n)));
```

Comment:
and here?

Reply:
same as above, `this.single` is already called in the line above